.gitignore

```diff
@@ -5,6 +5,7 @@
 build
 vendor
 node_modules
+vuln
```

Comment:
Previously, when generating the cpe sqlite file, we would store temporary files in /tmp. However, we are crawling multiple package repositories and need to store a lot of data. I ran into issues during development where tmp would fill up. I decided to change this to `vuln` in the current working directory. This is probably not ideal, but we currently only run generate from the nvd github workflow.